conf/application.conf

```diff
@@ -19,6 +19,7 @@ application.router=prod.Routes
 application.global=uk.gov.hmrc.nisp.config.ApplicationGlobal
 
 application.session.httpOnly=true
+play.filters.headers.contentSecurityPolicy= "default-src 'self' 'unsafe-inline' localhost:9032 www.google-analytics.com data:"
```

Comment:
9032 is the local port that serves the application.css from assets_frontend.
I don't see needing any other use-cases.